change/@ni-nimble-components-d6494d37-cc3a-4756-9bb7-c4a8ea4d89c7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Looks like all references of `eslint-disable-next-line @typescript-eslint/no-loop-func` would be cleaned up with this helper. Could we address those in the PR?